docs/howtos/extending/magicgui.md

```diff
@@ -450,6 +458,7 @@ You can create multiple layers by returning a list of
 {class}`~napari.layers.Layer`.
 
 ```python
+from magicgui import magicgui
 from typing import List
```

Comment:
```suggestion
```
napari in 0.6.0 is dropping python 3.9 support, so while we're here we should delete this import of List and the modify line 456/465 to use `list[napari.layers.Layer]`